src/elonacore.cpp

```diff
@@ -6403,7 +6403,7 @@ void initialize_adata()
     p = mdata_t::map_id_t::test_world;
     adata(16, p) = p;
     adata(15, p) = 0;
-    adata(0, p) = mdata_t::map_type_t::world_map;
+    adata(0, p) = mdata_t::map_type_t::field;
```

Comment:
I don't understand why you changed this value. The previous value `world_map` is correct, I think.

Reply:
Yes, might have been an extraneous change.